service/history/queuev2/pending_task_tracker.go

```diff
@@ -99,10 +105,15 @@ func (t *pendingTaskTrackerImpl) GetPendingTaskCount() int {
 	return len(t.taskMap)
 }
 
+func (t *pendingTaskTrackerImpl) GetPerDomainPendingTaskCount() map[string]int {
+	return t.taskCountPerDomain
```

Comment:
is this map going to be accessed concurrently? if so you might wanna return a copy of it here

Reply:
no, the whole component is not accessed concurrently